setup.py

```diff
@@ -25,8 +25,8 @@ def miniaudio_test_suite():
         zip_safe=False,
         include_package_data=False,
         py_modules=["miniaudio"],
-        install_requires=["cffi>=1.12.0", "wheel"],
-        setup_requires=["cffi>=1.12.0", "wheel"],
+        install_requires=["cffi>=1.12.0"],
+        # setup_requires=["cffi>=1.12.0", "wheel"],
```

Comment:
why was `wheel` removed here?

Reply:
The packages required for setup should be moved to pyproject.toml. setup.py should only be concerned with installation requirements